lib/log.go

```diff
@@ -137,16 +136,13 @@ func (m *LogMethods) Logbook(p *RefListParams, res *[]LogEntry) error {
 	}
 	ctx := context.TODO()
 
-	ref, err := repo.ParseDatasetRef(p.Ref)
+	ref, _, err := m.inst.ParseAndResolveRef(ctx, p.Ref, "local")
```

Comment:
Should the dsref package formalize the non-multiaddr sources into constants ("local", "registry")?

Reply:
I think it should be either the empty string for local resolution or a multiaddr. It's a temporary issue that the registry comes back as a URL that we should get fixed ASAP, but we'll need some URL-to-multiaddr translation work